webapp/frontend/PHP/exam/add/add-file.php

```diff
@@ -0,0 +1,47 @@
+<?php
+session_start();
+
+use navbar\NavBar;
+
+require("../../classes/NavBar.php");
+NavBar::userIsLogged(2);
+?>
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>Frontend</title>
+    <base href="/">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="icon" type="image/x-icon" href="favicon.ico">
+
+    <script src="../../css/bootstrap/js/bootstrap.bundle.min.js"></script>
```

Comment:
@MateuszKepczynski can we use CDNs to include bootstrap? I see that in couple of places you are using CDNs like: https://github.com/cvlab-ai/splinter/pull/53/files#diff-48fda346cd305c4de23e71ebdb0202025a5c730a5caf22719af50715227d43a9R29
but elsewhere you're using local files. 
Can we switch to using CDNs and remove local bootstrap files? - **please amend your commits to remove it from history.**

https://www.bootstrapdash.com/blog/use-bootstrap-with-php